### PR TITLE
NEP-180 Timely notification branch

### PR DESCRIPTION
**Pull request by syuraj**
-   [x] Include Jira Story No. in Title
-   [ ] Write Unit Tests
-   [ ] Add screenshots whenever possible
-   [x] Make sure Github Action build succeed
-   [ ] Address Automated Code Reviews from LGTM and Deepscan
-   [x] Add Reviewers
-   [ ] Force Merge PRs if not approved for testing in time

### CODE REVIEW

File: src/jobs/NotificationTrigger/notificationTime.js [hunk 1]
```diff
@@ -1,12 +1,17 @@
 const moment = require('moment')
 const momentTz = require('moment-timezone')
-const { NOTIFICATION_END_TIME, NOTIFICATION_START_TIME } = require('./config')
-const verifyNoticiableTime = (currentTime, startTime = NOTIFICATION_START_TIME, endTime = NOTIFICATION_END_TIME) => {
-	const currentNumericTime = Number(currentTime.replace(':', ''))
-	if (currentNumericTime >= startTime && currentNumericTime <= endTime) {
-		return true
-	}
-	return false
+const NOTIFICATION_TIMES = ['6:00','9:00','12:00','15:00','18:00','21:00'];
```
Comment (syuraj) [inline src/jobs/NotificationTrigger/notificationTime.js:3]: @ShivaAryal Can you fetch the times from environment variables?

File: src/jobs/NotificationTrigger/pushNotificationSender.js [hunk 2]
```diff
@@ -18,6 +17,7 @@ const sendPushNotification = async (notification) => {
 }
 
 const makeRequest = async (data = {}) => {
+	data.direct_book_ok = true;
```
Comment (syuraj) [inline src/jobs/NotificationTrigger/pushNotificationSender.js:20]: It is a good idea, but seems it requires some dependency in the app.
https://firebase.google.com/docs/cloud-messaging/android/receive#receive_fcm_messages_in_direct_boot_mode
@ShivaAryal

File: src/jobs/NotificationTrigger/pushNotificationSender.js [hunk 3]
```diff
@@ -1,10 +1,9 @@
 const logger = require('../../config/logger')
 const axios = require('axios')
 const { FIREBASE_SERVER_KEY } = require('./config')
-
 const sendPushNotification = async (notification) => {
 	try {
-		const response = await makeRequest(notification)
+		const response = await makeRequest(notification);
```
Comment (syuraj) [inline src/jobs/NotificationTrigger/pushNotificationSender.js:6]: @ShivaAryal just fyi, we try to avoid commas as it is not necessary

**Comment by syuraj** [comment]
@ShivaAryal can you also create/fix unit tests? 
On diff note: It might be easier for you to change your github email to siristechnology email.

File: src/jobs/NotificationTrigger/notificationTime.js (hunk 1 above)
Comment (ShivaAryal) [reply src/jobs/NotificationTrigger/notificationTime.js]: Yeah done

**Pull request merged**